Yoast/wordpress-seo
# Use StyledSection for analyses

## Summary

This PR can be summarized in the following changelog entry:

* Wraps the analysis inside a `yoast-components` `StyledSection`.

## Relevant technical choices:

* Created a activateTab action & reducer that gets used in the tabManager.
* Due to React we no longer need the HTML element ids. 

## Test instructions

This PR can be tested by following these steps:

* There are 2 analysis fields: Under readability & under a keyword.
* There are 2 different types: Under post / pages and a different one for taxonomies.
* Verify all these work exactly the same without this PR as with.
* Verify that the Yoast SEO plugin settings to enable / disable the analysis still work.

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/8784
